### PR TITLE
Spree Guides: Update logic customization examples

### PR DESCRIPTION
**Pull request by antoine-khawa**
Prepend customization only if needed

### CODE REVIEW

**Review by damianlegawiec — APPROVED** [review]

File: guides/src/content/developer/customization/logic.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ module MyStore
   end
 end
 
-::Spree::ProductsController.prepend MyStore::Spree::ProductsControllerDecorator
+::Spree::ProductsController.prepend MyStore::Spree::ProductsControllerDecorator if ::Spree::ProductController.included_modules.exclude?(MyStore::Spree::ProductControllerDecorator)
```
Comment (damianlegawiec) [inline guides/src/content/developer/customization/logic.md:83]:
```suggestion
::Spree::ProductsController.prepend MyStore::Spree::ProductsControllerDecorator if ::Spree::ProductsController.included_modules.exclude?(MyStore::Spree::ProductsControllerDecorator)
```

File: guides/src/content/developer/customization/logic.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ module MyStore
...
```
Comment (damianlegawiec) [inline guides/src/content/developer/customization/logic.md:57]:
```suggestion
::Spree::ProductsController.prepend MyStore::Spree::ProductsControllerDecorator if ::Spree::ProductsController.included_modules.exclude?(MyStore::Spree::ProductsControllerDecorator)
```

**Review by damianlegawiec — COMMENTED** [review]
typos :)

File: guides/src/content/developer/customization/logic.md (hunk 1 above)
Comment (sun-meihouwang) [reply guides/src/content/developer/customization/logic.md]: sorry, fixed

**Pull request merged**